src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -223,7 +223,7 @@ const MetaTraderConfig = (() => {
                 response.binary_transaction_id,
             ]),
             prerequisites: () => new Promise((resolve) => {
-                if (Client.get('is_virtual')) {
+                if (Client.get('is_virtual') || Client.get('landing_company_shortcode') === 'iom') {
                     resolve(needsRealMessage());
```

Comment:
we want to keep the same message for MX clients? it doesn't make too much sense, since MX is a real-money account